jbpm-services/jbpm-executor/src/main/java/org/jbpm/executor/impl/ExecutorServiceImpl.java

```diff
@@ -152,6 +153,7 @@ public void init() {
     		try {
 		        executor.init();
 		        this.executorStarted = true;
+		        ServiceRegistry.get().register(org.kie.api.executor.ExecutorService.class.getSimpleName(), this);
```

Comment:
Why not to use newly added ServiceRegistry.EXECUTOR_SERVICE also here?

Reply:
replaced